src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ControllerActionDescriptorProvider.cs

```diff
@@ -95,14 +94,14 @@ public void OnProvidersExecuted(ActionDescriptorProviderContext context)
             }
         }
 
-        protected internal IEnumerable<ControllerActionDescriptor> GetDescriptors()
+        internal IEnumerable<ControllerActionDescriptor> GetDescriptors()
```

Comment:
Nice. Makes it *very* clear external subclasses are no longer supported. Same for later `virtual` removals in this PR.